client/contexts/ServerContext/methods.ts

```diff
@@ -121,6 +121,7 @@ export type ServerMethods = {
 	'setUsername': (...args: any[]) => any;
 	'setUserPassword': (...args: any[]) => any;
 	'toggleFavorite': (...args: any[]) => any;
+	'updateEphemeralRoom': (...args: any[]) => any;
```

Comment:
It would be good if you could type this :)

Reply:
Sorry, can you please explain?

Reply:
Oh, i was just referring that it would be good to add typescript types to this 😬 but a lot of other functions have the same signature so no probs haha